remo/events/api/views.py

```diff
@@ -18,3 +30,62 @@ def retrieve(self, request, pk):
         serializer = EventDetailedSerializer(event,
                                              context={'request': request})
         return Response(serializer.data)
+
+
+class EventsKPIFilter(django_filters.FilterSet):
```

Comment:
This is a subset of the filters required in the public api view. I suggest we implement for the KPIs for now and extend it or revisit it when we tackle the public API filters.